src/routes/preHandlers/bills.js

```diff
@@ -92,3 +93,18 @@ exports.authorizeBillsCreation = async (req) => {
 
   return null;
 };
+
+exports.authorizeBillCreation = async (req) => {
+  const { credentials } = req.auth;
+  const companyId = credentials.company._id;
+
+  const customer = await Customer.countDocuments({ _id: req.payload.customer, company: companyId });
+  if (!customer) throw Boom.forbidden();
+
+  for (const bi of req.payload.billingItemList) {
```

Comment:
je ferrai plutot un .find( _id: { $in : req.payload.billingItemList}, company: companyId}). puis un check sur le nombre

Reply:
J'ai fait un peu différemment, dis moi ce que tu en penses